src/components/PlaybackControls/index.tsx

```diff
@@ -217,7 +218,9 @@ const PlayBackControls = ({
 
     return (
         <div className={styles.container}>
-            {minimalControls ? minimalEmbeddedControls : fullControls}
+            {PlaybackControlsDisplay.Full
```

Comment:
```suggestion
            {displayType === PlaybackControlsDisplay.Full
```
Small to tweak to ensure we are evaluating the prop as its not a boolean.